troposphere/static/js/components/login/LoginScreen.jsx

```diff
@@ -65,7 +65,7 @@ export default React.createClass({
     },
     attemptOpenstackLogin: function(username, password, projectName, provider, onLoginError) {
```

Comment:
I was trying to keep the call signature intact as other places depend on it.  This way I localize the changes and make this easier to test.